File: internal/content/testdata/bench/csharp/nth-fibonacci-list.cs
using System.Collections.Generic;

public class Solution {
    public long[] fibonacci_list(long n) {
        if (n == 0) return new long[0];
        if (n == 1) return new long[] { 0 };
        var result = new List<long> { 0, 1 };
        for (long i = 2; i < n; i++) {
            result.Add(result[(int)(i - 1)] + result[(int)(i - 2)]);
        }
        return result.ToArray();
    }
}

File: internal/content/testdata/bench/java/nth-fibonacci-list.java
class Solution {
    public long[] fibonacci_list(long n) {
        if (n == 0) return new long[0];
        long[] result = new long[(int)n];
        result[0] = 0;
        if (n == 1) return result;
        result[1] = 1;
        for (int i = 2; i < (int)n; i++) {
            result[i] = result[i - 1] + result[i - 2];
        }
        return result;
    }
}
